PSScriptAnalyzerRules.psd1

```diff
@@ -0,0 +1,84 @@
+@{
+    Severity     = @('Error')
+    IncludeRules = @(
+        'PSUseCompatibleSyntax',
+        'PSAvoidUsingCmdletAliases',
+        'PSAvoidUsingWriteHost',
+        'PSAvoidDefaultValueSwitchParameter',
```

Comment:
```suggestion
        'PSAvoidUsingCmdletAliases',
        'PSAvoidDefaultValueSwitchParameter',
```
I know that the use of `Write-Host` is not recommended, but this may still be the best option.